apps/console/.env.local

```
@@ -2,7 +2,7 @@
 #  Dev Server Configs
 # ----------------------------------------------------------------------------------
 DEV_SERVER_PORT=9001
-DEV_SERVER_HOST=localhost
+DEV_SERVER_HOST=127.0.0.1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DEV_SERVER_HOST key should go before the DEV_SERVER_PORT key

(UnorderedKey)